Makefile

```diff
@@ -6,6 +6,7 @@ pytest:
 
 type:
 	pytype -j auto
+	mypy ${LINT_PATHS}
```

Comment:
I would also add a `mypy` entry (would make debugging easier as pytype is quite slow as soon as you change one file)

Reply:
I would also activate the color output as we do for pytest, see https://github.com/python/mypy/issues/7771

Reply:
> I would also add a `mypy` entry (would make debugging easier as pytype is quite slow as soon as you change one file)

Should I replace `type` by `pytype` then?

Reply:
type is still fine as we want to run both in the CI or when developping.

Reply:
> I would also activate the color output as we do for pytest, see [python/mypy#7771](https://github.com/python/mypy/issues/7771)

I'm not sure to use `MYPY_FORCE_COLOR=1` properly, it does not seem to work as expected.

Reply:
yep, I saw that :/ will see if I can find an example online

Reply:
What about this @araffin?

Reply:
I double check and you seem to be doing things right...